packages/main/test/pages/MultiInput.html

```diff
@@ -258,7 +258,7 @@ <h1>Tokens + Suggestions</h1>
 	</div>
 
 	<div class="sample-container">
-		<ui5-multi-input show-suggestions id="token-unique" style="width: 50%">
+		<ui5-multi-input show-suggestions id="token-unique" show-value-help-icon style="width: 50%">
```

Comment:
the same comment as in the input test page.

Reply:
I am just extending the sample